### PR TITLE
Fix dead links in documentation

### PR DESCRIPTION
**Pull request by eXeC64**
This is for trac ticket [#11680](http://trac.mantidproject.org/mantid/ticket/11680).

This pull request either fixes or removes most of the broken links in the documentation.

**Testing**:
- Build the documentation
- Go to the pages affected
- Check the links work

### CODE REVIEW

File: Code/Mantid/docs/source/algorithms/ApplyCalibration-v1.rst [hunk 1]
```diff
@@ -15,8 +15,7 @@ updated as absolute positions and so this update can be repeated.
 The PositionTable must have columns *Detector ID* and *Detector
 Position*. The entries of the *Detector ID* column are integer referring
 to the Detector ID and the enties of the *Detector Position* are
-`V3Ds <http://www.mantidproject.org/V3D>`_ referring to the position of the detector whose ID is in
-same row.
+V3Ds referring to the position of the detector whose ID is in same row.
```
Comment (peterfpeterson) [inline Code/Mantid/docs/source/algorithms/ApplyCalibration-v1.rst]: I think this should point at `mantid.kernel.V3D` or just `V3D` directly. I'm not sure which.

File: Code/Mantid/docs/source/algorithms/LoadRaw-v3.rst [hunk 2]
```diff
@@ -35,7 +35,7 @@ and the different periods will be output as separate workspaces, which
 after the first one will have the period number appended (e.g.
 OutputWorkspace\_period). Each workspace will share the same
 `Instrument <http://www.mantidproject.org/Instrument>`__, SpectraToDetectorMap and
-`Sample <http://www.mantidproject.org/Sample>`__ objects. If the optional 'spectrum' properties are
+sample objects. If the optional 'spectrum' properties are
```
Comment (peterfpeterson) [inline Code/Mantid/docs/source/algorithms/LoadRaw-v3.rst]: This should reference `api/python/mantid/api/Sample.rst`

File: Code/Mantid/docs/source/algorithms/LoadRaw-v3.rst [hunk 3]
```diff
@@ -60,7 +60,7 @@ aspects of the output `Workspace <http://www.mantidproject.org/Workspace>`__:
    `Instrument <http://www.mantidproject.org/Instrument>`__.
 -  :ref:`algm-LoadLog` - Will look for any log files in the same
    directory as the RAW file and load their data into the workspace's
-   `Sample <http://www.mantidproject.org/Sample>`__ object.
+   sample object.
```
Comment (peterfpeterson) [inline Code/Mantid/docs/source/algorithms/LoadRaw-v3.rst]: This should reference `api/python/mantid/api/Sample.rst`

File: Code/Mantid/docs/source/algorithms/NormaliseByCurrent-v1.rst [hunk 4]
```diff
@@ -11,7 +11,7 @@ Description
 
 Normalises a workspace according to the good proton charge figure taken
 from the Input Workspace log data, which is stored in the workspace's
-`Sample <http://www.mantidproject.org/Sample>`_ object). Every data point (and its error) is divided
+sample object). Every data point (and its error) is divided
```
Comment (peterfpeterson) [inline Code/Mantid/docs/source/algorithms/NormaliseByCurrent-v1.rst]: This should reference `api/python/mantid/api/Sample.rst`

**Comment by peterfpeterson** [comment]
Please modify the lines I added comments to.

**Comment by AndreiSavici** [comment]
For DownloadFile one needs to change the documentation for Address property in the cpp file

**Comment by AndreiSavici** [comment]
the Align and focus powder link should point to http://198.74.56.37/ftp/external-data/MD5/e2b281817b76eadbc26a0a2617477e97 , but it need to be rewritten, as to one has to save the file with the name in the usage example. I've sent Vickie a request to update SaveLaueNorm. For InstrumentDefinitionFile, you need to talk to the GEM instrument scientist about the correct link

**Comment by AndreiSavici** [comment]
SaveLaueNorm: http://www.ccp4.ac.uk/newsletters/newsletter36/10_laue.html

**Pull request merged**